tests/core/config/checkers/test_data_node_config_checker.py

```diff
@@ -675,9 +675,10 @@ def test_check_exposed_types(self, caplog):
         assert len(Config._collector.errors) == 0
 
         config._sections[DataNodeConfig.name]["default"].properties = {"exposed_type": "modin"}
-        Config._collector = IssueCollector()
-        Config.check()
-        assert len(Config._collector.errors) == 0
+        with pytest.raises(SystemExit):
+            Config._collector = IssueCollector()
+            Config.check()
+        assert len(Config._collector.errors) == 1
```

Comment:
I think we should raise deprecation warning message instead?

Reply:
Yes, that's probably better.